provider/godaddy/godaddy.go

```diff
@@ -32,16 +33,20 @@ import (
 
 const (
 	gdMinimalTTL = 600
-	gdCreate     = iota
```

Comment:
what is the reason of changing it? IMHO this is more go-style

Reply:
The 3 constants **gdCreate, gdUpdate, gdDelete** are associated with the following variable **actionNames** used in code

`
log.Errorf("Unable to apply change %s on record %s, %v", actionNames[e.action], change, err)`

So I prefer explicit values in this case for readability.